rstudio/rstudio
# Fix labels on Global Options dialog to match what they were before localization

### Intent

Addresses #10381

NOTE: A couple of the property files were saved using Windows line ending; I changed that back to Linux, so you'll want to ignore whitespace when reviewing.

Labels on most controls in Global Options / Code preferences UI were changed as a result of the loc work (#10249). We want the English text to stay the same, so this addresses that.

### Approach

Controls that previously had custom labels (different than what is defined in the preferences schema) now have those labels again. For French, I just reused the existing labels from the preferences schema so this should be a no-op in French (we aren't currently testing French, just FYI).

### Automated Tests

None

### QA Notes

Verify that the English text shown in Global Options / Code (Editing/Display/Saving/Completion/Diagnostics matches what it was before the loc change for preferences was applied ().

You can also check all other Global Preferences UI panes; I didn't find label differences outside the Code pane, but if you do, please open a new issue.

The last build with the old labels was `2022.02.0-daily-357`. The first one with the incorrect labels was `2022.02.0-daily-359`.

### Checklist
